src/main/java/racegame/CarGame.java

```diff
@@ -0,0 +1,44 @@
+package racegame;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CarGame {
+    public static final int CHECK_NUMBER = 4;
+    private List<Car> carList;
+
+    public CarGame(List<Car> carList) {
+        this.carList = carList;
+    }
```

Comment:
```suggestion
    public CarGame(List<Car> carList) {
        this.carList = new ArrayList<>(carList);
    }
```
방어적 복사를 적용해보는 것은 어떨까요?

참고자료
- [방어적 복사](https://velog.io/@rito/%EB%B0%A9%EC%96%B4%EC%A0%81-%EB%B3%B5%EC%82%AC-vs-Unmodifiable-Collection)